src/compas_rhino/install.py

```diff
@@ -106,11 +115,45 @@ def install(version=None, packages=None):
         if status != 'OK':
             exit_code = -1
 
+    if exit_code == 0 and len(installed_packages):
+        print()
+        print('Running post-installation steps...')
+        print()
+        if not _run_post_execution_steps(after_rhino_install(installed_packages)):
+            exit_code = -1
+
     print('\nCompleted.')
     if exit_code != 0:
         sys.exit(exit_code)
 
 
+def _run_post_execution_steps(steps_generator):
+    post_execution_errors = []
+    for result in steps_generator:
+        if isinstance(result, Exception):
+            post_execution_errors.append(result)
+            continue
+
+        for item in result:
+            try:
+                package, message, success = item
+                status = 'OK' if success else 'ERROR'
+                print('   {} {}: {}'.format(package.ljust(20), status, message))
+            except ValueError:
+                post_execution_errors.append(ValueError('Step successful, but result is wrongly formatted: {}'.format(str(item))))
+
+    if post_execution_errors:
+        print()
+        print('One or more errors occurred:')
+        print()
+        for error in post_execution_errors:
+            print('   - {}'.format(repr(error)))
+
+        return False
```

Comment:
This is confusing to me.  If there are `post_execution_errors`, then it's been printed that the step was successful, but now we are getting an -1 exit code.  So is it a success or an error?

Reply:
How about now? I fixed the issue you found [here](https://github.com/compas-dev/compas/pull/770#discussion_r575369400) and hopefully made it a bit clearer what the return value of this method is supposed to be: https://github.com/compas-dev/compas/pull/770/commits/ae01ee4c4c6ae4fd106ead408391a734abbe432f

Reply:
I guess it's that I don't like the message of the `ValueError`. If the result isn't formatted correctly how can it be assume that the step was successful?

Reply:
ahh... mmm... fair point. how about: `ValueError('Step ran without errors but result is wrongly formatted: {}'.format(str(item))`?

Reply:
that is less confusing message for me.

Reply:
done! https://github.com/compas-dev/compas/pull/770/commits/4836330d3c59e811e31607c18dcdf92b23e319c7